forChanna.scd

```diff
@@ -2,36 +2,36 @@
     {
         s.options.memSize = 128 * 1024;
         s.options.sampleRate = 48000;
-	// ~libPath = "~/Documents/SC/2019/channa/forChannaHorwitz/lib/".asAbsolutePath; // change as needed
-	    ~libPath = "~/forChannaHorwitz/lib/".asAbsolutePath;
-        ~piecePath = ~libPath ++ "pieces/";
+	~libPath = "~/Documents/SC/2019/channa/forChannaHorwitz/lib/".asAbsolutePath; // change as needed
+
+        ~piecePath = ~libPath ++ "views/players/";
         ~pieces = ();
 
         PathName.new(~piecePath).files.collect({|f|
             f.fullPath.load
         }).do({|p|
             p.init;
-            ~pieces.add(p.title -> p)
+		    ~pieces.add(p.title -> p)
```

Comment:
Indent one level less.

Reply:
will fix

Reply:
Weird one because I don't think there's strong consensus on style in sclang, but I'd generally add only one level of indentation per nested multiline Function, so lines 11-14 would be:

```
        }).do({|p|
	    p.init;
	    ~pieces.add(p.title -> p)
        });
```

What do you think?